src/lib/services/simulator.ts

```diff
@@ -292,6 +292,12 @@ export class SimulatorService implements ISimulatorService {
     return version
   }
 
+  public async isLocalnetRunning(): Promise<boolean> {
+    const genlayerContainers = await this.getGenlayerContainers();
+    const runningContainers = genlayerContainers.filter(container => container.State === "running");
```

Comment:
@epsjunior this could break easily if the number of containers changes. Soon, we are removing two of them so we might want to look for a better way to understand this.
Maybe their names in an array in the config? So we ensure those exact containers are running?